cpmonitor/templates/city.html

```diff
@@ -1,179 +1,137 @@
 {% extends "pageframe.html" %}
```

Comment:
I think, this is quite difficult to summarize like that. And it would be kind of duplicate. We could upload Stefans PDF to the issue for future reference.